src/components/account/AccountLogin.tsx

```diff
@@ -54,24 +60,44 @@ const useStyles = makeStyles(theme => ({
   },
 }))
 
-const AccountLogin: FunctionComponent<AccountLoginProps> = ({isArcSignIn}) => {
+const AccountLogin: FunctionComponent<AccountLoginProps> = ({
+  isArcSignIn,
+  submitUsernameAndPassword,
+  isLoadingLoginWithUsernameAndPassword,
+  errorMessageLoginWithUsernameAndPassword,
+}) => {
   //const [alertMsg, setAlertMsg] = useState<{msg: string; type: AlertProps['severity']} | undefined>()
-
+  const sourceApp = useSourceApp()
   const classes = useStyles()
 
+  const featureToggles = useFeatureToggles<FeatureToggles>()
+  const createSignInTextBox = (text: string, sx?: BoxProps['sx']) => {
+    return (
+      <Box className={clsx(classes.text, !isArcSignIn && classes.mtbText)} sx={sx}>
+        {text}
+      </Box>
+    )
+  }
+
   return (
     <>
       <Container component="main" maxWidth="xs" className={classes.buttonContainer}>
-        <Box className={clsx(classes.text, !isArcSignIn && classes.mtbText)}>
-          {isArcSignIn
-            ? 'Please sign in to ARC using your Synapse account.'
-            : 'Please sign in to Open Bridge using your Synapse account.'}
-        </Box>
+        {createSignInTextBox(`Please sign in to ${sourceApp.longName} with your Synapse account: `)}
         <div className={classes.paper}>
           <Button variant="contained" href={Utility.getRedirectLinkToOneSage()} className={classes.arcSubmitbutton}>
             <SynapseLogo />
           </Button>
         </div>
+        {featureToggles['USERNAME PASSWORD LOGIN'] && (
```

Comment:
Use the feature toggle to conditionally display the username password login form.